tests/test_livingspace.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+import nose
+
+from models.livingspace import LivingSpace
+
+class TestLivingSpace(unittest.TestCase):
+	"""Test cases for the Living Space class"""
+
+	def setUp(self):
+		"""Setup a new Living Space object to be used for testing"""
+		self.livingspace = LivingSpace('turquoise')
+
+	def test_room_instance(self):
+		"""Tests if the object created is an instance of the LivingSpace class"""
+		self.assertIsInstance(self.livingspace,LivingSpace)
+
+	def test_room_type(self):
+		"""Tests if the object created is of type LivingSpace"""
+		self.assertEqual(type(self.livingspace),LivingSpace)
+
+	def test_room_is_empty(self):
+		"""Tests if the Living Space initially has no occupant"""
+		self.assertEqual(self.livingspace.occupant,[])
+
+	def test_room_number_of_occupants(self):
+		"""Tests if the Living Space is empty"""
+		self.assertEqual(self.livingspace.current_number,0)
+
+	def test_room_name(self):
+		"""Tests if the Living Space name is equal to 'turquoise' """
+		self.assertEqual(self.livingspace.name,'turquoise')
+
+	def test_room_type_variable(self):
+		"""Tests if the room type assigned to the room is 'LIVINGSPACE' """
+		self.assertEqual(self.livingspace.room_type,'LIVINGSPACE')
+
+	def test_room_max_occupant(self):
+		"""Tests if the maximum number of occupants in a Living Space is 6"""
+		self.assertEqual(self.livingspace.max_no_occupants,4)
```

Comment:
This method's docstring says that the maximum number of occupants allowed is 6 yet you are testing that the maximum number set is 4. Please review that.